utility/src/test/java/org/oppia/android/util/logging/EventBundleCreatorTest.kt

```diff
@@ -133,6 +176,118 @@ class EventBundleCreatorTest {
     assertThat(bundle).string("priority").isEqualTo("optional")
   }
 
+  @Test
+  fun testFillMetricsBundle_eventWithDefaultedContext_fillsDetailsAndRetsUnknownContext() {
```

Comment:
``eventWithDefaultloggableMetric``? There's no "context" field name in this proto.